### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by shunsukew**
CI
- tests
- format check
- build check

Also fixed format warnings to pass CI

### CODE REVIEW

**Comment by codespool** [comment]
I think you should rebase this to master, as it's showing the diff from https://github.com/AstarNetwork/swanky-cli/pull/1

File: src/commands/account/create.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Command, Flags } from "@oclif/core";
-import { yellowBright, greenBright } from "chalk";
+import yellowBright from "chalk";
+import greenBright from "chalk";
```
Comment (codespool) [inline src/commands/account/create.ts:3]: why split the import?

File: src/commands/account/list.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import { Command, Flags } from "@oclif/core";
-import { yellowBright, greenBright } from "chalk";
+import { Command } from "@oclif/core";
+import yellowBright from "chalk";
+import greenBright from "chalk";
```
Comment (codespool) [inline src/commands/account/list.ts:3]: same as above

File: src/commands/account/create.ts (hunk 1 above)
Comment (shunsukew) [reply src/commands/account/create.ts:3]: If you dot't add "chalk" in package.json dependencies https://github.com/AstarNetwork/swanky-cli/pull/5/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R32, getting lint error `error "chalk" is extraneous` which is against `unicorn/import-style` and `node/no-extraneous-import` rules.

I added chalk as a dependency, then with `v5.0.1` which is the latest, even build fails due to below
```
src/commands/account/create.ts:2:24 - error TS2614: Module '"chalk"' has no exported member 'greenBright'. Did you mean to use 'import greenBright from "chalk"' instead?

2 import { yellowBright, greenBright } from "chalk";
```

Comment (codespool) [reply src/commands/account/create.ts:3]: ah, sorry, chalk definitely needs to be added, that's my mistake.
The thing is, v5 is not compatible with TS bundler:
https://github.com/chalk/chalk/releases/tag/v5.0.0
So please just add 4.7

Comment (shunsukew) [reply src/commands/account/create.ts:3]: It says chalk requires ts version higher than 4.7 and we use ^4.7.4.
So it's already compatible?

**Comment by codespool** [comment]
@shunsukew I've updated the packages and switched to default import of `chalk` (actually got a correct error message this time :) )
Please check

**Comment by shunsukew** [comment]
> @shunsukew I've updated the packages and switched to default import of `chalk` (actually got a correct error message this time :) ) Please check

Thank you!

**Review by codespool — APPROVED** [review]

**Pull request merged**